site-src/implementations.md

```diff
@@ -130,6 +132,13 @@ Kong is actively working on an implementation of the Gateway API in the [Kong Ku
 [kic]:https://github.com/kong/kubernetes-ingress-controller
 [kong-gw-guide]:https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/using-gateway-api/
 
+### NGINX Kubernetes Gateway
+
+[NGINX Kubernetes Gateway][nginx-kubernetes-gateway] is an open-source project that provides an implementation of the Gateway API using [NGINX][nginx] as the data plane. The goal of this project is to implement the core Gateway APIs -- Gateway, GatewayClass, HTTPRoute, TCPRoute, TLSRoute, and UDPRoute -- to configure an HTTP or TCP/UDP load balancer, reverse-proxy, or API gateway for applications running on Kubernetes. NGINX Kubernetes Gateway is currently under development and supports a subset of the Gateway API.
```

Comment:
Tiny nit: Do you mind wrapping these lines at ~80 chars?

Reply:
Withdrawing this nit because it looks like we haven't required this for other implementations.